docs/user-guide/system-admin/configuration/infinispan.md

```diff
@@ -12,14 +15,38 @@ The Infinispan configuration will be located inside the following module in `sta
 
 Keep in mind that the module version may vary depending on your JBoss version.
 
-### Configuration for Internal data caches
+### Hibernate Cache
+
+The following is the recommended configuration for the Hibernate cache:
+
+```xml
+...
+<cache-container name="hibernate" default-cache="local-query" jndi-name="java:jboss/infinispan/container/hibernate" start="EAGER" module="org.jboss.as.jpa.hibernate:4">
```

Comment:
Ah. It looks like the attribute `jndi-name="java:jboss/infinispan/container/hibernate"` isn't normally in `standalone.xml`, and this binding seems to be required by JndiInfinispanRegionFactory. Then the question is whether we should still be using JndiInfinispanRegionFactory.

@carlosmunoz Shouldn't Hibernate should work out of the box, on top of Infinispan? What are we doing that breaks that?